src/resources/vue/components/ItemCard.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="item-card">
+        <div class="left">
+            <Icon icon="book" size="90" color="white" />
+            <div class="info">
+                <p>{{ title }}</p>
+                <p>{{ description }}</p>
+            </div>
+        </div>
+        <div v-if="warning === 'true'" class="right">
+            <Icon icon="warning" size="35" />
+        </div>
+    </div>
+</template>
+
+<script>
+import Icon from "./icon.vue";
+
+export default {
+    name: "ItemCard",
+    props: {
+        title: String,
+        description: String,
+        warning: Boolean,
+    },
+    components: {
+    Icon,
+    Icon,
+    Icon
+},
+};
+</script>
+
+<style lang="scss">
+.item-card {
+    width: 667px;
+    height: 116px;
+    background: #1D243A;
+    border-radius: 8px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 13px;
+
+    .left {
```

Comment:
waarom al deze whitespace?

Reply:
De extra enters tussen de classes? Want dat is voor een beetje overzicht.